caddy/Caddyfile.example

```diff
@@ -0,0 +1,57 @@
+# Replace your-domain.com with your actual domain
+{$DOMAIN} {
+	import snippets/Logging
+
+	# Traffic Analytics service
+	import snippets/TrafficAnalytics
+
+	# ActivityPub Service
+	import snippets/ActivityPub
+
+	# Default proxy everything else to Ghost
+	handle {
+		reverse_proxy ghost:2368
+	}
+
+	# Optional: Enable gzip compression
+	encode gzip
+
+	# Optional: Add security headers
+	import snippets/SecurityHeaders
+}
+
+# If you have a separate domain for Ghost Admin uncomment the below block
+# {$ADMIN_DOMAIN} {
+#     import snippets/Logging
+#
+# 	# Traffic Analytics service
+# 	import snippets/TrafficAnalytics
+#
+# 	# ActivityPub Service
+# 	import snippets/ActivityPub
+#
+# 	# Default proxy everything else to Ghost
+# 	handle {
+# 		reverse_proxy ghost:2368
+# 	}
+#
+# 	# Optional: Enable gzip compression
+# 	encode gzip
+#
+# 	# Optional: Add security headers
+# 	import snippets/SecurityHeaders
+# }
+
+# If you want to redirect the www variant of your domain to the non-www varient uncomment the below 4 lines
+# Note: You must have DNS setup correctly for this to work
+# www.{$DOMAIN} {
+# 	import snippets/Logging
+# 	redir https://{$DOMAIN}{uri}
+# }
+
+# If you want to redirect the non-www variant of your domain to the www varient uncomment the below 4 lines and replace REPLACE_ME with your domain
+# Note: You must have DNS setup correctly for this to work
+# REPLACE_ME {
+# 	import snippets/Logging
+# 	redir https://REPLACE_ME{uri}
+# }
```

Comment:
This redirect is a requirement for ActivityPub - I think Ghost in general expects this redirect. I think this should be uncommented by default, and instructions for the DNS setup given 🤔

Reply:
I'm confused, which part specifically is a requirement? This part at the bottom is for redirecting when you have a base domain of eg `www.my-domain.com` and want to redirect `my-domain.com` -> `www.my-domain.com`

Reply:
I've added a comment and will make sure its added to the docs 👍